sql/core/src/test/scala/org/apache/spark/sql/execution/command/v2/AlterTableAddPartitionSuite.scala

```diff
@@ -38,7 +39,8 @@ class AlterTableAddPartitionSuite
       val errMsg = intercept[AnalysisException] {
         sql(s"ALTER TABLE $t ADD PARTITION (id=1)")
       }.getMessage
-      assert(errMsg.contains(s"Table $t does not support partition management"))
+      val tableName = UnresolvedAttribute.parseAttributeName(t).map(quoteIdentifier).mkString(".")
+      assert(errMsg.contains(s"Table $tableName does not support partition management"))
```

Comment:
Use `checkError`. This will allow to modify the error messages in the JSON files by tech editors w/o modifying the internal Spark tests.